torch/csrc/autograd/variable.h

```diff
@@ -518,9 +518,6 @@ inline Variable make_variable(
     at::Tensor data,
```

Comment:
What is this function supposed to do now? The comment seems out of data as well.

Reply:
I'm planning in a follow up to delete all occurrences of `make_variable`. But I think in the current codebase it kind of plays the same role as the legacy `Variable` constructor in Python.